examples/audio_server/server.cc

```diff
@@ -0,0 +1,132 @@
+#include <memory>
+#include <mutex>
+
+#include "../../blocks/http/api.h"
+#include "../../bricks/dflags/dflags.h"
+#include "../../bricks/sync/waitable_atomic.h"
+
+DEFINE_uint16(port, 8080, "The local port to use.");
+DEFINE_uint32(n, 3, "Max number of audio streams");
+
+const std::string cmd_kill = "kill";
```

Comment:
Why a string constant if you could use `enum class`, or even a special `[unordered]set` for the IDs to be killed?